libs/design/src/scss/theming/contrast/luminance/luminance.scss

```diff
@@ -6,10 +6,15 @@
 // and "1" corresponding to "full brightness".
 //
 // See https://www.w3.org/TR/WCAG20-TECHS/G17.html#G17-tests
+
 @function daff-luminance($color) {
-	$red: nth($linear-channel-values, red($color) + 1);
-	$green: nth($linear-channel-values, green($color) + 1);
-	$blue: nth($linear-channel-values, blue($color) + 1);
+	$red-color: red($color);
```

Comment:
Does this actually fail the linting rule? `green(color)` shouldn't.

Reply:
Yes, green(color), red(color), blue(color) all violated the added rule.